### DISCUSSION TITLE
How to do Img2Img with the library

### DISCUSSION THREAD

**Discussion by ThatGuySam** [andrei-zgirvaci/expo-stable-diffusion#9] [General]
I found out about a project recently that restores photos of stillborn infants so that parents get to see their child the way they were meant to be born. 

https://www.babyangelpics.com/

I know Stable Diffusion can do Img2Img on Hugging Face, and I was wondering how you could do the same with this library so that you could make an app that gives parents photos of their children from the ones taken from the birth while letting the parents keep their privacy.

**Comment by andrei-zgirvaci** [comment]
Hi, `ml-stable-diffusion` actually supports image2image generation using the [VAEEncoder.mlmodelc (image encoder model)](https://github.com/apple/ml-stable-diffusion#swift-package-details). However, I haven't had the time to add support for it in `expo-stable-diffusion`.

I am open to PRs in case you would like to work on this! 😊

P.S. I have converted the issue into a discussion.

**Comment by syeminpark** [comment]
Hello, thanks for the great work. Is  image2image generation still not supported in your library?
